aten/src/ATen/native/native_functions.yaml

```diff
@@ -8319,6 +8319,10 @@
   python_module: linalg
   variants: function
 
+- func: linalg_tensorsolve(Tensor self, Tensor input2, int[]? axes=None) -> Tensor
```

Comment:
"input2" -> "other" to be consistent with other PyTorch functions.

"axes" -> "dims" to be consistent with other PyTorch functions. Note that PyTorch's bindings should remap "axes->dims", too.

Reply:
What's the plan for out support?

Reply:
I wasn't thinking about out support, since NumPy doesn't do that. I guess it can be added to this PR.